rpy2/ipython/rmagic.py

```diff
@@ -533,10 +610,21 @@ def publish_graphics(self, graph_dir, isolate_svgs=True):
     @argument(
         '-i', '--input', action='append',
         help=textwrap.dedent("""
-        Names of input variable from `shell.user_ns` to be assigned to R
-        variables of the same names after using the Converter self.converter.
-        Multiple names can be passed separated only by commas with no
-        whitespace.""")
+        Names of Python objects to be assigned to R
+        objects after using the default converter or
+        one specified through the argument `-c/--converter`.
+        Multiple inputs can be passed separated only by commas with no
+        whitespace.
+
+        Names of Python objects can be either the name of an object
+        in the current notebook/ipython shell, or a path to a name
+        nested in a namespace visible from the current notebook/ipython
+        shell. For example, '-i myvariable' or
+        '-i mypackage.myothervariable' would both work.
+
+        Each input can be either the name of Python object, in which
+        case the same name will be used for the R object, or an
+        expression of the form <r-name>=<python-name>.""")
```

Comment:
An alternative syntax could be `<python name> as <r name>`; this might be a bit easier to understand without reading documentation.

Reply:
It would indeed by easy to read, however:
- ipython/jupyter magic arguments are like CLI arguments (and this would change that)
- this is relies on Python's `argparse` module for parsing arguments (and this would no longer make this possible).

For now it seems more practical and consist to use the `a=b` syntax.

Reply:
Sure, it makes sense. Just for completeness, another option would be using syntax like in walrus operator (`a:=b`) though I think that `a=b` is fine.